src/backends/pgfplotsx.jl

```diff
@@ -436,15 +436,20 @@ function pgfx_axis!(opt::PGFPlotsX.Options, sp::Subplot, letter)
 end
 # --------------------------------------------------------------------------------------
 # display calls this and then _display, its called 3 times for plot(1:5)
+function _create_backend_figure(plt::Plot{PGFPlotsXBackend})
+
+end
+
 function _series_updated(plt::Plot{PGFPlotsXBackend}, series::Series)
-    # TODO: don't rebuild plots so often
 end
 
+# TODO: don't rebuild plots so often
+# IDEA: use functor to only build plot once
 function _update_plot_object(plt::Plot{PGFPlotsXBackend})
-    plt.o = PGFPlotsX.GroupPlot()
+    plt.o = PGFPlotsX.TikzDocument()
```

Comment:
It might make sense (from a readability p.o.v) to do something like `td = PGFPlotsX.TikzDocument` and then only in the end set `plt.o = td`.

Reply:
Thanks for the review :)
I moved that now into a struct to avoid the recreation.